daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala

```diff
@@ -52,7 +52,7 @@ object Util {
 
   private val isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows")
 
-  private val daffodilRoot = sys.env.getOrElse("DAFFODIL_HOME", ".")
+  private val daffodilRoot = "."
```

Comment:
Can we delete daffodilRoot and it's uses? Looks like its only used in a couple of places, where a value of "." is essentially a no-op.

Reply:
Done